gfx/HollowMoon/pngs_normal_24x24/vehicle/seats/credits.txt

```diff
@@ -0,0 +1 @@
+hoasd1
```

Comment:
A

Reply:
Also Joshuu did the mounted spare wheel if you want to continue with credits and can you add vp_halfboard_cover to the halfboard JSON as I forget about it?